tox.ini

```
@@ -57,6 +57,8 @@ deps =
     oldestdeps: zeep<3.5.0
     oldestdeps: h5netcdf<0.9
     oldestdeps: cdflib<0.3.20
+    oldestdeps: pytest<5.5
```

```suggestion
    oldestdeps: pytest<6.0
```
let us see if this works?

We should also set this version in setup.cfg.